backend/functions/src/course/get_course_id.ts

```diff
@@ -30,7 +30,7 @@ function constructUrl(courseCatalogName, roster = 'FA21') {
   return url
 }
 
-async function getCourseId(courseCatalogNameRaw, roster = 'FA21') {
+async function getCourseId(courseCatalogNameRaw: string, roster: any) {
```

Comment:
Can probably say `roster: string` because with the default argument from before, we know it's a string like `'FA21'`